CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## Version 2.2.2 - 2020/07/30
```

Comment:
FWIW, we could format dates according to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)…

Reply:
So `YYYY-MM-DD`?